webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const path = require('path');
+
+module.exports = {
+  entry: "./src/index.js",
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: "bundle.js"
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader"
+        }
+      }
+    ]
+  }
+};
```

Comment:
You may have to add in rules for css later or it could possibly create a compiling error.